cedar-policy/src/api.rs

```diff
@@ -2577,6 +2835,49 @@ impl Context {
     /// if attributes have the wrong types (e.g., string instead of integer).
     /// Since different Actions have different schemas for `Context`, you also
     /// must specify the `Action` for schema-based parsing.
+    /// ```
+    /// use cedar_policy::{Context, RestrictedExpression, Schema};
+    /// use std::collections::HashMap;
+    /// use std::str::FromStr;
+    /// # use cedar_policy::{Entities, EntityId, EntityTypeName, EntityUid, Request,PolicySet};
```

Comment:
let's hide these imports
```suggestion
    /// # use cedar_policy::{Context, RestrictedExpression, Schema};
    /// # use std::collections::HashMap;
    /// # use std::str::FromStr;
    /// # use cedar_policy::{Entities, EntityId, EntityTypeName, EntityUid, Request,PolicySet};
```